chips/stm32f4xx/src/fsmc.rs

```diff
@@ -144,13 +144,13 @@ struct FsmcBank {
 }
 
 const FSMC_BANK1: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x60000000 as *const FsmcBank) };
-const FSMC_BANK2_RESERVED: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x0 as *const FsmcBank) };
+// const FSMC_BANK2_RESERVED: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x0 as *const FsmcBank) };
 const FSMC_BANK3: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x68000000 as *const FsmcBank) };
-const FSMC_BANK4_RESERVED: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x0 as *const FsmcBank) };
+// const FSMC_BANK4_RESERVED: StaticRef<FsmcBank> = unsafe { StaticRef::new(0x0 as *const FsmcBank) };
```

Comment:
Should these be labeled BANK0..BANK3 instead of BANK1..BANK4 to match the indicies in the array in which they are contained? It seems odd that programming the bank labeled BANK1 requires passing a `bank_id` of `0` to `write_data()`

Reply:
I added an enum for the Banks instead of using an index.